tests/requirements.txt

```
@@ -3,7 +3,8 @@
 anaconda-client
 beautifulsoup4
 chardet
-conda >=4.13
+conda >= 4.13
```

```suggestion
conda >=4.13
```
IIRC matchspec parsing can be brittle, was this change intentional?

---

No